build/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Convox Build'
```

Comment:
Just a general comment on YAML. You don't need to put string values inside quotes unless they begin with a number.